manifests/admin/install.pp

```diff
@@ -0,0 +1,8 @@
+# Pulp Admin Install Packages
+class pulp::admin::install {
+  package { ['pulp-admin-client',]: ensure => $pulp::admin::version }
+
+  if $pulp::admin::puppet {
+    package { ['pulp-puppet-admin-extensions',]: ensure => $pulp::admin::version }
+  }
+}
```

Comment:
Nitpick, but my own preference, and the style in our modules tends to be without an array for single packages, and a new line after the resource title, e.g.

```
resource { 'title':
  param1 => value,
  param2 => value,
}
```